reconcile/utils/slack_api.py

```diff
@@ -114,24 +119,36 @@ def get_user_id_by_name(self, user_name):
             raise UserNotFoundException(result['error'])
         return result['user']['id']
 
-    def get_channels_by_names(self, channels_names):
-        return {k: v['name'] for k, v in self._get('channels').items()
+    def get_channels_by_names(self, channels_names, limit=CHANNELS_LIMIT):
```

Comment:
since your next step would be to implement this through app-interface, it would probably make sense to decide the limits inside `_get` (i suppose the limits will be in `self`)